### PR TITLE
Update to scala 2.11

### PR DESCRIPTION
**Pull request by hermanbanken**
I have changed the scala version to 2.11 and made everything compile again. All tests succeed too.

### CODE REVIEW

**Comment by dwins** [comment]
Thanks for taking the time to put together a PR.  I'll take a minute this evening to re-run the tests and merge.  I should let you know I'm not really maintaining this code any more, but if you're interested in moving it forward I can give you commit access and maybe answer some questions if you have any.

**Comment by hermanbanken** [comment]
No problem. I just wanted to try some geo stuff, but was blocked by the
non-2.11 support, so I decided to help the project forward a little instead
of looking on or rolling by own. I'm not really interested in further major
maintenance, but I guess I might do some bug fixes when I find bugs...

Op wo 13 apr. 2016 om 22:47 schreef David Winslow <notifications@github.com

File: examples/src/main/scala/feature/GeoCrunch.scala [hunk 1]
```diff
@@ -39,7 +39,7 @@ trait GeoCrunch {
   ) = {
     val it = fc.features()
     try {
-      while (it.hasNext) try {
```
Comment (hermanbanken) [inline examples/src/main/scala/feature/GeoCrunch.scala]: What was your goal with this try @dwins? The compiler warned me that it did nothing without its own catch or finally clause. Whole loop is wrapped too.

**Comment by dwins** [comment]
Ok, no problem. Like I said I'll merge this PR once I verify the tests pass (sure that won't be a problem) and I'm happy to merge any future ones if you provide them.

File: examples/src/main/scala/feature/GeoCrunch.scala (hunk 1 above)
Comment (dwins) [inline examples/src/main/scala/feature/GeoCrunch.scala]: Yep it's totally superfluous, possibly a leftover from a poorly-executed refactoring.  I agree with the decision to remove it.

**Pull request merged**